SwitchSpeak-iOS/SwitchSpeak-iOS/CrumbManager.swift

```diff
@@ -19,21 +19,29 @@ class CrumbStack {
         // i.e. "I", "am", etc.
         var label:UILabel
         
+        var scale:CGFloat
+        
         init(content:String) {
             label = UILabel()
             label.text = content
+            scale = 1
         }
         
         /*
          * Sets the color of the breadcrumb as well as its location on the screen
          * relative to the view containing it. The provided index corresponds
          * to the crumb's index in the stack holding it.
          */
-        func setLocationAndColor(byIndex index:Int) {
-            label.frame = CGRect(x: 10 + (index % 5) * 80, y: 40 * (index / 5 + 1), width: 80, height: 40)
+        
+        func setLocationAndSize(index i:Int, width w:CGFloat) {
```

Comment:
Can you rewrite the function header to read
`func setLocationAndSize(atIndex index:Int, withWidth width:CGFloat) {`
Swift wants parameter labels to clarify what we're feeding our functions, and it is recommended to use these labels like natural language (https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/Methods.html)

Reply:
In general we also want to use parameter names longer than one character, and we want them (like all variables) to be as descriptive as possible, even when it's obvious.